### PR TITLE
encriptacion en window

### PR DESCRIPTION
**Pull request by lueimg**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -9,9 +9,12 @@ var express = require('express'),
     path = require('path'),
     multipart = require('connect-multiparty'),
     fs = require('fs'),
-    mime = require('mime');
-
+    mime = require('mime'),
+    bcrypt = require('bcrypt'),
+    port = 1234;
```
Comment (mvergaray) [inline server.js]: Where is port supposed to be used?

File: server/features/migrateUserPassword.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            printLog(err);
+            return db.rollback(function () {
+              throw err;
+            });
+          }
+          db.commit();
+          res.status(200).json({ results: { list:rows } });
+        });
+
+
```
Comment (mvergaray) [inline server/features/migrateUserPassword.js]: unnecessary line

File: server/features/users.js [hunk 3]
```diff
@@ -224,12 +226,14 @@ rutas.route('/')
...
-        ' password = "' + req.body.password + '"  ' +
+    var hash = bcrypt.hashSync(req.body.password, 10),
+        query = 'UPDATE USERS SET ' +
+        ' password = "' + hash + '"  ' +
         ' where id = ' + req.user.id;
 
-    if (req.body.oldPassword !== req.user.password) {
-      res.status(200).send({code: 200, message: 'Su clave actual no coincide'});
+    if (!bcrypt.compareSync(req.body.oldPassword, req.user.password)) {
+      res.status(200).send({code: 200, message: 'Su clave actual no coincide, Por favor asegurese de ' +
```
Comment (mvergaray) [inline server/features/users.js]: coincide. Por favor, asegurese

File: uploads/read.txt [hunk 4]
```diff
@@ -1 +0,0 @@
-En este folder se almacenarán temporalmente los archivos que se suban al servidor.
```
Comment (mvergaray) [inline uploads/read.txt]: Why removing this file? When I tried uploading a new empty folder, it was not submitted if it was empty, that's why I created that file. Won't it affect?

Comment (lueimg) [inline uploads/read.txt]: ohh sorry , I did't know

File: server/features/migrateUserPassword.js (hunk 2 above)
Comment (lueimg) [inline server/features/migrateUserPassword.js]: done

**Pull request merged**